File: src/main/java/com/bajdcc/LALR1/interpret/os/user/routine/URNews.java
package com.bajdcc.LALR1.interpret.os.user.routine;

import com.bajdcc.LALR1.interpret.os.IOSCodePage;
import com.bajdcc.util.ResourceLoader;

/**
 * 【用户态】新闻
 *
 * @author bajdcc
 */
public class URNews implements IOSCodePage {
	@Override
	public String getName() {
		return "/usr/p/news";
	}

	@Override
	public String getCode() {
		return ResourceLoader.load(getClass());
	}
}

File: src/main/kotlin/com/bajdcc/LALR1/interpret/os/user/routine/URNews.kt
package com.bajdcc.LALR1.interpret.os.user.routine

import com.bajdcc.LALR1.interpret.os.IOSCodePage
import com.bajdcc.util.ResourceLoader

/**
 * 【用户态】新闻
 *
 * @author bajdcc
 */
class URNews : IOSCodePage {
    override val name: String
        get() = "/usr/p/news"

    override val code: String
        get() = ResourceLoader.load(javaClass)
}
